linera-core/src/unit_tests/client_tests.rs

```diff
@@ -1953,6 +1889,9 @@ where
     Ok(())
 }
 
+// TODO: this test is currently intermittently failing if the faulty validators respond to
```

Comment:
Please create an issue for it and cite it here.
We're trying to have no naked `TODO`s in the code, only `TODO(#1234)`.